lib/HeroBonus.h

```diff
@@ -215,6 +215,7 @@ class CSelector : std::function<bool(const Bonus*)>
 	BONUS_NAME(ADDITIONAL_UNITS) /*val of units with id = subtype will be added to hero's army at the beginning of battle */\
 	BONUS_NAME(SPOILS_OF_WAR) /*val * 10^-6 * gained exp resources of subtype will be given to hero after battle*/\
 	BONUS_NAME(BLOCK)\
+	BONUS_NAME(NO_TERRAIN_PENALTY) /* subtype - terrain type */\

```

Comment:
What worse it's don't break save loading in same time.
Though something else would break due to changes in ENUM...